httpc.go

```diff
@@ -144,6 +146,20 @@ func (r *Request) ClientCertificatesFromFiles(certFile, keyFile, caFile string)
 	return r.ClientCertificates(clientCert, clientKey, caCert)
 }
 
+// ClientCertificatesFromInstance sets the client certificates from a cert instance
+func (r *Request) ClientCertificatesFromInstance(clientCertWithKey tls.Certificate, caChain []*x509.Certificate) (*Request, error) {
+	tlsConfig, err := setupClientCertificate(clientCertWithKey, caChain, r.client.Transport.(*http.Transport).TLSClientConfig)
+
```

Comment:
I know, super nitpicky, but can we maybe compactify this, dropping the extra empty line (same at the end of the function) to keep code formatting consistent?